config/index.js

```diff
@@ -22,28 +22,28 @@ function logDirExistsSync() {
 var config = {
   local: {
     mode: 'local',
-    level: 'debug',
+    level: 'fatal',
```

Comment:
hmm ok i suppose that works for now. I wish we had a better solution to this though.